main.go

```diff
@@ -232,15 +246,19 @@ func run(args []string, stdin io.Reader, stderr io.Writer) error {
 
 	var errors []error
 	for _, filename := range files {
-		f, err := parser.ParseFile(fset, filename, nil /* src */, parser.AllErrors|parser.ParseComments)
+		// getting the content of original file
+		content, err := ioutil.ReadFile(filename)
```

Comment:
The ioutil package is deprecated. We can use `os.ReadFile` here instead. https://pkg.go.dev/os#ReadFile